pkg/console/operator/sync_v400.go

```diff
@@ -332,10 +332,10 @@ func SyncRoute(co *consoleOperator, operatorConfig *operatorv1.Console) (*routev
 
 	// we will not proceed until the route is valid. this eliminates complexity with the
 	// configmap, secret & oauth client as they can be certain they have a host if we pass this point.
-	if len(rt.Spec.Host) == 0 {
-		// TODO STATUS
+	host := routesub.GetCanonicalHost(rt)
```

Comment:
This is still the first attempt in the sync loop to get the `host`, so the above 2 instances of `GetCanonicalHost()` will not be called if this one does not pass.